vuu-ui/packages/vuu-table-extras/src/column-formatting-settings/ColumnFormattingPanel.tsx

```diff
@@ -123,3 +115,20 @@ export const ColumnFormattingPanel = ({
     </div>
   );
 };
+
+function formattingSettingsByColType(props: FormattingSettingsProps) {
+  const { column } = props;
+
+  if (isDateTimeColumn(column)) {
```

Comment:
somewhere we ought to warn the user if they do something invalid, like assigning a date/time type to a column with a serverDatatype that is not `long`.  This is part of the weird dissonance arising out of the fact that we're mixing server assigned data types and client side config.

Maybe we ignore for now and assume we'll fix all this when we overhaul the metadata sent by server.